src/boot/meson.build

```diff
@@ -11,8 +11,7 @@ libefitest = static_library(
                 'efi-string.c',
         ),
         build_by_default : false,
-        include_directories : [
-                basic_includes,
+        include_directories : basic_includes + [
```

Comment:
Those changes are meh. Meson flattens the lists always, so it doesn't care, and the form with `+` is more involved and harder to read.